src/accountsdb/buffer_pool.zig

```diff
@@ -19,26 +21,75 @@ comptime {
     _ = offset;
 }
 
+const FrameRef = packed struct(u32) {
+    f_idx: FrameIndex,
+    hit: bool,
+    const INIT = FrameRef{
+        .f_idx = INVALID_FRAME,
+        .hit = false,
+    };
+};
+
+const Rc = std.atomic.Value(u32);
```

Comment:
do we need this alias? -- throughout the codebase we usually rename `const Atomic = Value` and then use `Atomic(u32)` throughout -- would make `frame_ref_counts: []Atomic(u32)` much easier to read -- wdyt

Reply:
Removed alias, replaced with Atomic(u32) https://github.com/Syndica/sig/commit/f40689214b7f8a0945d24c5a2847c29af6db0203